test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -361,6 +361,12 @@ TEST_F(QuicPlatformTest, QuicLog) {
 #define VALUE_BY_COMPILE_MODE(debug_mode_value, release_mode_value) debug_mode_value
 #endif
 
+TEST_F(QuicPlatformTest, LogEndOfLine) {
+  GetLogger().set_level(ERROR);
+  QUIC_DLOG(ERROR) << "aaaa" << std::endl;
```

Comment:
Does this emit an extra empty new line compared to `QUIC_DLOG(ERROR) << "aaaa";`?

If yes, can we strip the ending new lines in quic logging impl?

Reply:
done

Reply:
Nice, thanks!